lib/views/home/widgets/bottom_sheet/menu/menu_bottom_sheet_widget.dart

```diff
@@ -15,6 +16,35 @@ class MenuBottomSheetWidget extends ConsumerWidget {
 
   @override
   Widget build(BuildContext context, WidgetRef ref) {
+    var status = ref.watch(notificationPermissionStatusProvider);
+
+    return status.when(
+      skipLoadingOnRefresh: false,
+      skipLoadingOnReload: false,
+      data: (data) {
+        var isNotificationMenuVisibility = true;
```

Comment:
should be "visible" rather than "visibility"

Reply:
Fixed